blindsign/blindrsa/brsa.go

```diff
@@ -1,13 +1,28 @@
-package blindrsa
-
-// This package implements the blind RSA protocol based on the CFRG specification:
-// https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-rsa-blind-signatures
+// Package blindrsa implements the RSA Blind Signature Protocol.
+//
+// The RSA Blind Signature protocol, also called RSABSSA
```

Comment:
```suggestion
// The RSA Blind Signature protocol, and its variant RSABSSA
```

might be an overlook, but the RFC intro is the following
```
   This document specifies (1) a protocol for computing RSA blind
   signatures using RSA-PSS encoding and (2) a family of variants
   (Section 5) for this protocol, denoted RSABSSA (RSA Blind Signature
   with Appendix).  In order to facilitate deployment, it is defined in
   such a way that the resulting (unblinded) signature can be verified
   with a standard RSA-PSS library.
```

my understanding is RSA Blind Signature protocol is more general than RSABSSA, which is a category of variants for the protocol